### ISSUE TITLE
[AIR] TensorFlow warns to use `distribute.MultiWorkerMirroredStrategy` when I'm already using it

### ISSUE THREAD

**Issue by bveeramani** [ray-project/ray#25140]
### What happened + What you expected to happen

I'm training a TensorFlow image classifier and I get a warning:

```
(BaseWorkerMixin pid=9064) Instructions for updating:
(BaseWorkerMixin pid=9064) use distribute.MultiWorkerMirroredStrategy instead
(BaseWorkerMixin pid=9064) 2022-05-24 02:35:12.986754: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
```
I'm already using `MultiWorkerMirroredStrategy`, though, so I'm not sure why I'm getting the warning.

```
def train_loop_per_worker(config):
    dataset_shard = train.get_dataset_shard("train")
    strategy = tf.distribute.experimental.MultiWorkerMirroredStrategy()
    with strategy.scope():
        model = build_model()
        model.compile(optimizer='adam',
                    loss=tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True),
                    metrics=['accuracy'])

    ...
```
### Versions / Dependencies

**TensorFlow**: 2.9.1
**Ray**: 5e4225237e0806e3300381b4f8f39494c67e2906

### Reproduction script

Run the example in #24633.

### Issue Severity

Low: It annoys or frustrates me.

**Comment by cszhu** [comment]
Hello! This P2 issue has seen no activity in at least 2 years. It will be closed in 2 weeks as part of ongoing cleanup efforts.

Please remove the `pending-cleanup` label if you believe this issue should remain open.

Thanks for contributing to Ray!
